### DISCUSSION TITLE
Firebase admin SDK Dart vm

### DISCUSSION THREAD

**Discussion by hmbenhaim** [firebase/flutterfire#8618] [Feature request]
Hi I wanted to know if there is any plans to develop Firebase admin SDK for dart VM, before I developed flutter apps I developed for the web and I used Firebase packages for the UI and my backend was node js and I used the admin SDK for that, since 2019 I migrated the apps to flutter and it was amazing as a js developer I developed ui and backend in the same language which is also possible in dart which is amazing, but it missing one really important thing Firebase admin SDK so for firestore and storage I use googleapis package by Google which is ok but not close to the experience of the admin SDK but for Auth I couldn't find any satisfied package, to complete the experience developing dart app with flutter and dart backend I think is important,
Thank you 🙏

**Comment by MichaelVerdon** [comment]
Closing at this is a duplicate. https://github.com/firebase/flutterfire/discussions/9526
